modules/apps/object/object-web/src/main/resources/META-INF/resources/js/utils/fdsSourceDataRenderer.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+
+interface fdsSourceDataRenderer {
+	itemData: {
+		system: boolean;
+	};
+}
+
+export default function fdsSourceDataRenderer({
+	itemData,
```

Comment:
@pedro-oliveira446, you can use here the [Destructuring Assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) to make the code easier to read, with this you no longer need to use `itemData.system`, just the `system` will be enotgh: like:

```tsx
itemData: {system}
```